src/app.ts

```diff
@@ -32,7 +34,17 @@ export default async function spawn(config: Config = {}): Promise<Express> {
   /**
    * App Routes
    */
+  // Home page
   app.get("/", home.index);
 
+  // Winkels
+  app.get("/winkels", winkels.getStores);
+  app.get("/winkels/:storeId", winkels.getStore);
+  app.post("/winkels", winkels.addStore);
```

Comment:
De conventie die ik gewend ben voor route-methoden is de volgende:
- **index** om een lijst op te halen (zoals `/winkels/`)
- **show** om een item op te halen met een id (`/winkels/:id`)
- **new** om een formulier te krijgen om een nieuw item aan te maken (`/winkels/new`)
- **create** om een item aan te maken met een `POST /winkels/`
- **edit** om een formulier te krijgen om een item aan te passen (`/winkels/:id/edit`)
- **update** om  een item aan te passen met een `PUT /winkels/:id`
- **destroy** om een item weg te smijten met een `DELETE`

Is dat een conventie die wij willen gebruiken? Of blijven we bij `getStore`, `addStore`, ...? @LouiseDck @zoeparman